### PR TITLE
List employees feature added

### PR DESCRIPTION
**Pull request by MohamedHalith001**
- [x] List employees feature added

### CODE REVIEW

File: src/main/java/in/mohamedhalith/model/Employee.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package in.mohamedhalith.model;
+
+public class Employee {
+	private String name;
+	private int id;
+	private int sickLeave = 2;
+	private int casualLeave = 2;
+	private int earnLeave = 1;
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/mohamedhalith/model/Employee.java]: earnedLeave

File: src/main/webapp/index.jsp [hunk 2]
```diff
@@ -1,20 +1,12 @@
-<%@page import="java.util.Date"%>
-<%@page import="java.time.LocalDate"%>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt"%>
-<html>
+<!DOCTYPE html>
+<html lang="en">
 <head>
 <title>MyApp</title>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Change the title

**Review by ghost — CHANGES_REQUESTED** [review]
Added review comments

File: src/main/java/in/mohamedhalith/model/Employee.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	public int getCasualLeave() {
+		return casualLeave;
+	}
+	public void setCasualLeave(int casualLeave) {
+		if(casualLeave <0 || casualLeave >3) {
+			throw new IllegalArgumentException("Invalid no. of casual leaves");
+		}
+		this.casualLeave = casualLeave;
+	}
+	public int getEarnLeave() {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/mohamedhalith/model/Employee.java]: Change getter method.

**Review by ghost — CHANGES_REQUESTED** [review]
Comments Added

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**